versioned_docs/version-v1.3/vm/virtual-machines.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 1
+sidebar_label: Virtual Machines
+title: "Virtual Machines"
+keywords:
+  - virtual machine
+  - VM
+  - guest operating system
+  - guest OS
+description: Information concerning virtual machines that run on top of the Harvester cluster
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.4/vm/virtual-machines"/>
+</head>
+
+You can create [Linux VMs](../vm/create-vm.md) using one of the following methods: 
+
+- Harvester UI: On the **Virtual Machines** screen, click **Create** and configure the settings on each tab. 
+- Kubernetes API: Create a `VirtualMachine` object. 
+- [Harvester Terraform Provider](../terraform/terraform-provider.md): Define a `harvester_virtualmachine` resource block. 
+
+Creating [Windows VMs](../vm/create-windows-vm.md) on the Harvester UI involves slightly different steps. Harvester provides a VM template named `windows-iso-image-base-template` that adds a volume with the Virtio drivers for Windows, which streamlines the VM configuration process. If you require Virtio devices but choose to not use the template, you must add your own Virtio drivers for Windows to enable correct hardware detection. 
+
+## Supported Guest Operating Systems
+
+The following operating systems have been validated to run in Harvester virtual machines:
+
+| Operating System | Harvester v1.3.0 | Harvester v1.3.1 | Harvester v1.3.2 | Harvester v1.4.0 |
```

Comment:
We should cover only the verified OS versions for each Harvester minor version rather than duplicating all versions in each Harvester version document. For example, in 1.3, we should not have the version info for 1.4. 

cc @jillian-maroket